website/docs/r/datadog_tagrules.html.markdown

```diff
@@ -0,0 +1,89 @@
+---
+subcategory: "Datadog"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_datadog_monitor_tagrule"
+description: |-
+  Manages Tag Rules on the datadog Monitor.
+---
+
+# azurerm_datadog_monitor_tagrule
+
+Manages TagRules on the datadog Monitor.
+
+## Example Usage
+
+### Adding TagRules on monitor
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-datadog"
+  location = "West US 2"
+}
+resource "azurerm_datadog_monitor_tagrule" "example" {
+  datadog_monitor_id = azurerm_datadog_monitor.example.id
+  log {
+    subscription_log_enabled = true
+  }
+  metric {
+    filtering_tag {
+      name   = "Test"
+      value  = "Logs"
+      action = "Include"
+    }
+  }
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `datadog_monitor_id` - (Required) The Datadog Monitor Id which should be used for this.
+
+* `resource_group_name` - (Required) The name of the Resource Group where the datadog Monitor should exist.
+
+---
+
+* `rule_set_name` - (Optional) The name of the Tag Rules configuration.
+
+* `log` - (Optional) A `log_rules` block as defined below.
+
+* `metric` - (Optional) A `metric_rules` block as defined below.
+
+---
+
+An `log` block supports the following:
+
+* `aad_log_enabled` - (Optional) Boolean flag specifying if AAD logs should be sent for the Monitor resource.
+
+* `subscription_log_enabled` - (Optional) Boolean flag specifying if Azure subscription logs should be sent for the Monitor resource.
+
+* `resource_log_enabled` - (Optional) Boolean flag specifying if Azure resource logs should be sent for the Monitor resource.
+
+* `filtering_tag` - (Optional) A `filtering_tag` block as defined below.
```

Comment:
Can we just rename this to `filter` given the current resource is a tag rule?

Reply:
Done